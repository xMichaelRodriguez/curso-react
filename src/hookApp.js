import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h3>Hola mundo</h3>
        </div>
    )
}
