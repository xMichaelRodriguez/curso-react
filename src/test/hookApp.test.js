import { shallow } from "enzyme";
import React from "react";

import { HookApp } from "../hookApp";

describe("Retorna match de componente", () => {


  test("Muestra el componente HookApp", () => {


    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
