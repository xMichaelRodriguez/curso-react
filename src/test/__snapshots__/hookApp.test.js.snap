// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Retorna match de componente Muestra el componente HookApp 1`] = `
<div>
  <h3>
    Hola mundo
  </h3>
</div>
`;
